wpilibc/src/main/native/cpp/shuffleboard/SendableCameraWrapper.cpp

```diff
@@ -1,47 +1,34 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "frc/shuffleboard/SendableCameraWrapper.h"
+#include <functional>
```

Comment:
Shouldn’t this still include the SendableCameraWrapper header?  You could conditionally implement the functions that use cscore in the header...

Reply:
I just changed this to include the header, and used the same trick as ShuffleboardContainer to make it work properly.